spark/src/main/antlr/FlintSparkSqlExtensions.g4

```diff
@@ -76,6 +77,39 @@ dropCoveringIndexStatement
     : DROP INDEX indexName ON tableName
     ;
 
+materializedViewStatement
+    : createMaterializedViewStatement
+    | showMaterializedViewStatement
+    | describeMaterializedViewStatement
+    | dropMaterializedViewStatement
+    ;
+
+createMaterializedViewStatement
+    : CREATE MATERIALIZED VIEW (IF NOT EXISTS)? mvName=multipartIdentifier
```

Comment:
Why not calling it VIEW ?
So it would be compatible with ANSI SQL ?

Reply:
did you mean logical view? I assume CREATE VIEW is non-materialized view in SQL